packages/packages.csv

```diff
@@ -71,7 +71,7 @@ Microsoft.AspNetCore.Mvc.Localization,ship,include
 Microsoft.AspNetCore.Mvc.Razor,ship,include
 Microsoft.AspNetCore.Mvc.Razor.Extensions,ship,include
 Microsoft.AspNetCore.Mvc.RazorPages,ship,include
-Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,exclude
+Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,include
```

Comment:
This doesn't have any runtime components. Are we just using this as a way to get the package in the `.All` package now?